### PR TITLE
test for crud-operations profile

### PR DESCRIPTION
**Pull request by BelousSofiya**

### CODE REVIEW

File: profiles/tests/test_crud_profile.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+            comp_is_startup=True,
+            comp_registered=False
+        )
+        cls.test_profile2 = Profile.objects.create(
+            person=cls.test_person2,
+            comp_registered=True,
+            comp_is_startup=False
+        )
+
+        Category.objects.create(category_id=1,
```
Comment (Shushunya) [inline profiles/tests/test_crud_profile.py:49]: "Producer" is a type of Activitty, not Category, as I understand. Better change that to `cheese` or something.

Comment (BelousSofiya) [reply profiles/tests/test_crud_profile.py:49]: Thank you. Done

File: profiles/views.py [hunk 2]
```diff
@@ -84,12 +86,27 @@ class ProfileDetail(RetrieveUpdateDestroyAPIView):
...
     permission_classes = (IsAuthenticated,)
 
-    def get_queryset(self):
+    def get_queryset(self, pk=None):
         user_id = self.request.user.id
         if self.request.method == "DELETE":
             return Profile.objects.filter(person_id=user_id, is_deleted=False)
         if self.request.method == 'GET':
             return Profile.objects.filter(is_deleted=False)
+        if self.request.method == 'PUT':
```
Comment (Shushunya) [inline profiles/views.py:95]: You can combine these two, since they return the same objects. I suggest `if self.request.method in ["PUT", "PATCH"]`

Comment (BelousSofiya) [reply profiles/views.py]: You are right. Thank you. Done.

**Review by mehalyna — APPROVED** [review]

**Pull request merged**